_data/cis-features.yml

```diff
@@ -0,0 +1,17 @@
+- title: Compliance modules
+  description: Exclusive access to a set of CIS-compliant wrapper modules.
+
+- title: CIS certified
+  description: The Gruntwork compliance modules are certified compliant by the Center for Internet Security.
```

Comment:
Oh, that's a big deal! We should use this exact phrasing on the home page too, in the `index/_learn-more.html` below.